src/plugins/workspace/public/components/workspace_form/workspace_permission_setting_panel.tsx

```diff
@@ -131,7 +131,16 @@ export const WorkspacePermissionSettingPanel = ({
 
   return (
     <div>
-      <EuiFlexGroup gutterSize="m">
+      <EuiFlexGroup gutterSize="s">
+        <EuiFlexItem style={{ maxWidth: 150 }}>
+          <EuiCompressedFormRow
+            label={i18n.translate('workspaceForm.permissionSetting.typeLabel', {
+              defaultMessage: 'Type',
+            })}
+          >
+            <></>
```

Comment:
Why having this? `<></>`?

Reply:
It seems that FormRow requires a children. I noticed that the part was written this way before, so I imitated the style in the newly added FormRow.